src/main/java/uk/gov/companieshouse/api/accounts/service/impl/DebtorsService.java

```diff
@@ -38,23 +40,40 @@
     private DebtorsTransformer transformer;
     private SmallFullService smallFullService;
     private KeyIdGenerator keyIdGenerator;
+    private DebtorsValidator debtorsValidator;
 
     @Autowired
     public DebtorsService(DebtorsRepository repository,
                           DebtorsTransformer transformer,
                           SmallFullService smallFullService,
-                          KeyIdGenerator keyIdGenerator) {
+                          KeyIdGenerator keyIdGenerator, DebtorsValidator debtorsValidator) {
 
         this.repository = repository;
         this.transformer = transformer;
         this.smallFullService = smallFullService;
         this.keyIdGenerator = keyIdGenerator;
+        this.debtorsValidator = debtorsValidator;
     }
 
     @Override
     public ResponseObject<Debtors> create(Debtors rest, Transaction transaction,
                                           String companyAccountsId, HttpServletRequest request)
-        throws DataException {
+        throws DataException, RestException {
+
+        Errors errors;
+
+        try {
+            errors = debtorsValidator.validateDebtors(rest, transaction);
+        } catch (RestException restException) {
+            throw restException;
```

Comment:
It doesn't make any sense to catch and then throw the exact same exception? It would have the exact same result as removing the try catch.